docs/rancher/virtualization-management.md

```diff
@@ -32,12 +32,36 @@ Please refer to the [Harvester & Rancher Support Matrix](https://www.suse.com/su
 <Tabs>
 <TabItem value="ui" label="UI" default>
 
+1. Check and prepare the container image.
+
+    Depends on the version of your Rancher server e.g. v2.7.9, a new POD named `cattle-cluster-agent-***` with container image `rancher/rancher-agent:v2.7.9` will be created on Harvester cluster to fulfill the importing task. This dynamic image is not packed into Harvester ISO and will be pulled on the fly. When your cluster is not directly internet reachable, please follow one of the options below:
+
+    - Set a private [containerd-registry](../advanced/settings.md#containerd-registry) and add the additional image. Harvester will pull the image from this registry automatically.
+
+    - If [http-proxy](../airgap.md/#configure-an-http-proxy-in-harvester-settings) is used, configure and confirm it is working as expected. Your configured [DNS servers](../install/update-harvester-configuration.md#dns-servers) should be able to resolve the domain name `docker.io`.
+
+    - Manually download the image (`docker pull rancher/rancher-agent:v2.7.9 && docker save -o rancher-agent.img rancher/rancher-agent:v2.7.9`), copy it to each node of your Harvester cluster and load it (`sudo -i docker image load -i rancher-agent.img`). Run `crictl image ls | grep "rancher-agent"` on each node to ensure the image is ready.
```

Comment:
I'd suggest using `ctr` command for loading images in containerd. `docker` is an alias to `nertdctl` and `nerdctl` is included just for convenience, it's not officially supported by SUSE or RKE2 team.

Reply:
Thanks @w13915984028. one tiny nit. Can we name `rancher-agent.img` to `rancher-agent.tar`? it's a tarball.